src/NHibernate/Transaction/AdoNetWithSystemTransactionFactory.cs

```diff
@@ -236,7 +236,7 @@ public virtual void Wait()
 					// Remove the block then throw.
 					Unlock();
 					throw new HibernateException(
-						"Synchronization timeout for transaction completion. Either raise {Cfg.Environment.SystemTransactionCompletionLockTimeout}, or this may be a bug in NHibernate.");
+						$"Synchronization timeout for transaction completion. Either raise {Cfg.Environment.SystemTransactionCompletionLockTimeout}, or this may be a bug in NHibernate.");
```

Comment:
Trivial bug fix by the way. I do not think it is even worth a mention in commit.